opentelemetry-api/src/opentelemetry/trace/propagation/__init__.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 from typing import Optional
 
-from opentelemetry.context import get_value, set_value
+from opentelemetry.context import attach, get_value, set_value
```

Comment:
What's this for?

Reply:
I think it got added as I first thought of placing the function here but also exporting it from trace module. Not sure why pylint/isort didn't catch this as an unused import. 🤔